ansible/cloud_providers/azure_destroy_env.yml

```diff
@@ -10,4 +10,7 @@
     - name: Run infra-azure-template-destroy
       include_role:
         name: infra-azure-template-destroy
+    - name: Remove the service principal
+      include_role:
+        name: infra-azure-create-service-principal
```

Comment:
i guess it should be `infra-azure-delete-service-principal` here ?

Reply:
You are correct, and I fixed the reference